### ISSUE TITLE
Documenter 1.0

### ISSUE THREAD

**Issue by ArnoStrouwen** [JuliaDocs/DocumenterTools.jl#80]
Could DocumenterTools be made compatible with Documenter 1.0?
At least one SciML documentation upgrade is blocked by this.

**Comment by mortenpi** [comment]
Out of curiosity, which one? I'm wondering why/how you're actually using it during a documentation build.

**Comment by ArnoStrouwen** [comment]
It is `Optimization.jl`.
I'm not familiar how/why it is used myself.

**Comment by mortenpi** [comment]
I looked at https://github.com/SciML/Optimization.jl/pull/594, but I didn't immediately see DocumenterTools anywhere?

Note: we need to update the compat here anyway, so this discussion is tangential to that.

**Comment by ArnoStrouwen** [comment]
I think I got it here:
https://github.com/SciML/Optimization.jl/actions/runs/6287593416/job/17072214034?pr=594#step:4:17

**Comment by mortenpi** [comment]
It didn't really make sense to me that DocumenterTools would even be in that manifest, but here's where it's coming from:

```
(docs) pkg> why DocumenterTools
  OptimizationSpeedMapping → SpeedMapping → DocumenterTools
```

Will PR SpeedMapping to have it removed, since it's not actually using it.

**Comment by Datseris** [comment]
+1 to this. Note that in JuliaDynamics **we genuinely use DocumenterTools**. Specifically, and as I have pointed out on Slack, we do:

```
using Documenter
using DocumenterTools: Themes
```

which no longer works. We get the error: `Themes` not defined.

See here: https://github.com/JuliaDynamics/ComplexityMeasures.jl/actions/runs/6300078885/job/17102094879#step:5:10

That's the package versions we get installed:

 https://github.com/JuliaDynamics/Attractors.jl/actions/runs/6281895507/job/17060755223?pr=94#step:4:27 ; Documetner Tools resolves to v0.1.2

However, Documenter resolves to version 1+: https://github.com/JuliaDynamics/Attractors.jl/actions/runs/6281895507/job/17060755223?pr=94#step:4:249


please, please do consider fixing DocumenterTools.jl. it would take so much effort for me to change 20 package's build, only to change it back again to documenter 1.0 later.

---

p.s.: I don't understand how https://github.com/NicolasL-S/SpeedMapping.jl/pull/14 would fix things, as the dependency there is 0.1. It is not blocking, the latest stable of DocumenterTools.jl is indeed 0.1.

**Comment by mortenpi** [comment]
> p.s.: I don't understand how https://github.com/NicolasL-S/SpeedMapping.jl/pull/14 would fix things, as the dependency there is 0.1. It is not blocking, the latest stable of DocumenterTools.jl is indeed 0.1.

Because that's the reason why it ends up in the `Optimization.jl` manifest. No DocumenterTools, no problem with Documenter 1.0. But, of course, it won't help you if you legitimately need to combine Documenter 1.0 with DocumenterTools.
